### DISCUSSION TITLE
Migration from Nuxt-3 to Nuxt-4?

### DISCUSSION THREAD

**Discussion by abhishekJ-RMS** [nuxt/nuxt#27711] [General]
I have some questions about the upcoming Nuxt 4 version. Currently, we have several apps running on Nuxt 2, and we are considering upgrading to Nuxt 3. However, since migrating from Nuxt 2 to Nuxt 3 is already a significant challenge, would it be more prudent to wait and migrate directly to Nuxt 4 instead?

I am also looking for a list of updates between Nuxt 3 and Nuxt 4 to help us avoid similar issues in the future if we need to migrate from Nuxt 3 to Nuxt 4.

**Comment by TheAlexLichter** [comment]
Nuxt 3 -> Nuxt 4 will be pretty straightforward. You can focus on your Nuxt 3 migration and do the Nuxt 4 changes when this is done ☺️
